From 51b782bb9b6270a0b1f69df8fe9e2e4f5aead2a2 Mon Sep 17 00:00:00 2001
Message-Id: <51b782bb9b6270a0b1f69df8fe9e2e4f5aead2a2.1616414932.git.avarab@gmail.com>
In-Reply-To: <20210306192525.15197-1-avarab@gmail.com>
References: <20210306192525.15197-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 5 Mar 2021 17:28:19 +0100
Subject: [PATCH v4 4/4] Makefile/coccicheck: set SPATCH_BATCH_SIZE to 8
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the "xargs -n X" limit for running spatch from 1 to 8, as
suggested by Jeff King[1]. Now that we're not using --all-includes
anymore this is going to take much less memory, so bumping the limit
should be OK.

1. https://lore.kernel.org/git/YEIGzXMDax83cwAx@coredump.intra.peff.net/

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index eaac14275bb..543fe472482 100644
--- a/Makefile
+++ b/Makefile
@@ -1204,7 +1204,7 @@ SPATCH_FLAGS = --all-includes --include-headers-for-types --patch .
 # For the 'coccicheck' target; setting SPATCH_BATCH_SIZE higher will
 # usually result in less CPU usage at the cost of higher peak memory.
 # Setting it to 0 will feed all files in a single spatch invocation.
-SPATCH_BATCH_SIZE = 1
+SPATCH_BATCH_SIZE = 8
 
 # For the 'coccicheck' target; SPATCH_XARGS can be used to manually
 # tweak the xargs invocation. By default we invoke "xargs -n 1", and
-- 
2.31.0.285.gb40d23e604f

